pkg/agent/utils.go

```diff
@@ -808,3 +840,140 @@ func getCustomDataFromJSON(jsonStr string) string {
 	}
 	return customDataObj["customData"]
 }
+
+// GetOrderedKubeletConfigString returns an ordered string of key/val pairs
+// copied from AKS-Engine and filter out flags that already translated to config file
+func GetOrderedKubeletConfigString(k *api.KubernetesConfig, cs *api.ContainerService) string {
```

Comment:
nit: check if k != nil?

Reply:
it was checked in baker.go but I think we should check kubeletconfig though